doc/user/builders-writing.xml

```diff
@@ -654,7 +654,11 @@ def generate_actions(source, target, env, for_signature):
 
     <para>
 
-    The &generator; must return a
+    The required arguments must be listed in the function signature
+    in the correct order to be interpreted as a &generator;. More
+    arguments may be specified, but any extra required arguments must
+    be filled (such as by using a Python &partial; function or a &lambda;)
```

Comment:
These two are not defined entities in the xml, so fail the build.  Would suggest something like:

... such as by using a Python <function>functools.partial</function> function or a lambda expression.

Reply:
Ah, thank you for this. I am still working on doc updates and writing a test to demonstrate what I am fixing, so I'll roll this in with my next commit.